charts/descheduler/values.yaml

```diff
@@ -19,6 +19,7 @@ resources:
 nameOverride: ""
 fullnameOverride: ""
 
+cronJobApiVersion: "batch/v1"  # for k8s version < 0.21.0, use "batch/v1beta1"
```

Comment:
I don't think it's necessary to parameterize this, but if so we should make a note to remove this once `v1beta1` is removed as a TODO

Reply:
Added a TODO. Happy to remove and hard-code but I was hoping to honor the three release rule:
"a particular descheduler release is only tested against the three latest k8s minor versions" from README